brambling/templates/brambling/_event_list_item.html

```diff
@@ -13,7 +13,18 @@ <h5 class="datecard-month">{{ event.start_date|date:"M" }}</h5>
 			</div>
 		</div>
 	</a>
-	<h3>{{ event.name }}</h3>
+	<h3>
+		{{ event.name }}
+		{% if is_admin and not event.is_published %}
+			<small class='text-danger tipped' title="You can see this event because you're are an admin for it">
```

Comment:
I'm gonna change from danger to something else. Danger should only be for things that you can fix or really should not do.